StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Elements/CardSectionWithScanner/CVCRecollectionElement.swift

```diff
@@ -59,36 +81,54 @@ final class CVCRecollectionElement: Element {
 
     func beginEditing() {
         DispatchQueue.main.async {
-            self.cvcRecollectionView.textFieldElement.beginEditing()
+            self.textFieldElement.beginEditing()
         }
     }
 
     var validationState: ElementValidationState {
-        return cvcRecollectionView.textFieldElement.validationState
+        return textFieldElement.validationState
     }
     func clearTextFields() {
-        self.cvcRecollectionView.textFieldElement.setText("")
+        textFieldElement.setText("")
+    }
+
+    func updateErrorLabel() {
+        if case let .invalid(error, shouldDisplay) = textFieldElement.validationState, shouldDisplay {
```

Comment:
Why don't we do `cvcRecollectionView.errorLabel.isHidden = shouldDisplay`? Seems like `shouldDisplay` is unused?

Reply:
shouldDisplay is used, its value determines if we end up in this case or the else case. In plain english this line says:
```
If the validation state has an error and we should display it, ...
```

Reply:
Ah I missed the trailing `, shouldDisplay` on line 96, thanks!